### PR TITLE
fix(bootstrap): link librdkafka in C_INCLUDE_PATH and LIBRARY_PATH

### PR DESCRIPTION
**Pull request by mdtro**
I'm not entirely certain if this is the _right_ way to do this, but currently `make bootstrap` runs into issues with installing `confluent-kafka` without these values set. I had to set these environment variables to get things working.

```
fatal error: 'librdkafka/rdkafka.h' file not found
      #include <librdkafka/rdkafka.h>
               ^~~~~~~~~~~~~~~~~~~~~~
      1 error generated.
      error: command 'clang' failed with exit status 1
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  WARNING: No metadata found in ./.venv/lib/python3.8/site-packages
  Rolling back uninstall of confluent-kafka
  ```

### CODE REVIEW

File: scripts/lib.sh [hunk 1]
```diff
@@ -109,6 +109,8 @@ install-py-dev() {
         pip install https://storage.googleapis.com/python-arm64-wheels/psycopg2_binary-2.8.6-cp38-cp38-macosx_11_0_arm64.whl
         # The CPATH is needed for confluent-kakfa --> https://github.com/confluentinc/confluent-kafka-python/issues/1190
         export CPATH="$(brew --prefix librdkafka)/include"
+        export C_INCLUDE_PATH="$(brew info librdkafka | sed -n 4p | cut -d' ' -f1)/include/"
```
Comment (asottile) [inline scripts/lib.sh:112]: this isn't right -- this is what the brew prefix line above this is supposed to do

File: scripts/lib.sh [hunk 2]
```diff
@@ -109,6 +109,8 @@ install-py-dev() {
...
+        export LIBRARY_PATH="$(brew info librdkafka | sed -n 4p | cut -d' ' -f1)/lib"
```
Comment (asottile-sentry) [inline scripts/lib.sh:113]: this shouldn't be needed -- my guess is `make bootstrap` skips this part entirely (the `CPATH` already adds the include directory) -- also `brew info | sed` is very fragile -- and the entire reason `brew --prefix` exists!

Comment (mdtro) [reply scripts/lib.sh:113]: `brew --prefix` gave me a different result. I wasn't too sure which one I actually needed. I wasn't able to get it to work with `brew --prefix` on my system, but I'll retry it.

```
❯ brew --prefix librdkafka
/opt/homebrew/opt/librdkafka
```

```
❯ echo $C_INCLUDE_PATH
/opt/homebrew/Cellar/librdkafka/1.9.2/include/
```

Comment (asottile-sentry) [reply scripts/lib.sh:113]: yeah and `readlink -f /opt/homebrew/opt/librdkafka` will tell you the rest of the story

Comment (asottile-sentry) [reply scripts/lib.sh:113]: well, readlink doesn't work on macos -- but you get the idea -- brew uses symlinks into `Cellar`

**Pull request closed without merging**